Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C

```diff
@@ -71,12 +71,18 @@
 //   isRealData (bool)         = true/false for data/MC (default true)
 //   truncateFlag    (int)     = Flag to treat different depths differently (0)
 //                               both depths of ieta 15, 16 of HB as depth 1 (1)
-//                               all depths as depth 1 (2), all depths in HE
-//                               with values > 1 as depth 2 (3), all depths in
-//                               HB with values > 1 as depth 2 (4), all depths
-//                               in HB and HE with values > 1 as depth 2 (5),
-//                               for depth = 1 and 2, depth = 1, else depth
-//                                = 2 (6). (Default 0)
+//                               all depths as depth 1 (2), ignore all depth
```

Comment:
Is there any reason why the description of the truncateFlag is different here with respect to the one in the other macros?